### DISCUSSION TITLE
raspberry PI supported

### DISCUSSION THREAD

**Discussion by yang-dong-dong** [tauri-apps/tauri#2029] [Show and tell]
![image](https://user-images.githubusercontent.com/20069609/122707667-0d485d00-d28d-11eb-8489-a3c4e9005449.png)

**Comment by nothingismagick** [comment]
We have a discord discussion dedicated to this topic, perhaps we should move it here for more visibility. Look for the `#arm-platform` channel and scroll up. :)
